frontend/src/components/form/BookingsSearch.tsx

```diff
@@ -79,28 +80,13 @@ export default function Search({ getSearchResults }: { getSearchResults: Functio
                     <StyledFormLabel htmlFor='location'>
                       Location
                     </StyledFormLabel>
-                    <StyledNativeSelect
-                      input={
-                        <Field component={({ field, form, ...props }: any) => (
-                          <StyledSelectInput {...field} {...props} />
-                        )} />
-                      }
-                      /* TODO set default value as clerk location */
-                      id='location'
-                      name='location'
-                      variant='outlined'
-                      defaultValue='Victora'
-                    >
-                      <option value='Victoria'>Victoria</option>
-                      <option value='Nanaimo'>Nanaimo</option>
-                      <option value='Courtenay'>Courtenay</option>
-                      <option value='Abbotsford'>Abbotsford</option>
-                      <option value='Vancouver'>Vancouver</option>
-                    </StyledNativeSelect>
+                    
                     <ErrorMessage name='location' />
                   </StyledFormControl>
                 </Grid>
-
+                <CourtLocationSelect
+                  id='location'
+                  name='location'/>
```

Comment:
![image](https://user-images.githubusercontent.com/46575401/99616594-6c31b500-29d2-11eb-8fd8-8362563ed1b3.png)

please move this to  
```
<CourtLocationSelect
                  id='location'
                  name='location'/>
```

 top should fix the layout problem. Thanks!